study-builder/studies/brain/emails/exit_request.html

```diff
@@ -103,7 +103,7 @@
 <table style="width: 540px; height: 10px; padding: 0; -webkit-text-size-adjust: 100%;-ms-text-size-adjust: 100%; mso-table-lspace: 0pt; mso-table-rspace: 0pt; border-collapse: collapse !important;"
        border="0" cellpadding="0" cellspacing="0" width="100%">
   <tr style="width: 540px; height: 10px; padding: 0;">
-    <img src="https://storage.googleapis.com/brain-prod-assets/BrainCancerLine.png"
+    <img src="https://storage.googleapis.com/cmi-study-dev-assets/brain/project-line.png"
```

Comment:
same as above
why is this pointing to dev-assets rather than variable substitution like "${assetsBucketName}"/brain ??